app/models/group_order_article.rb

```diff
@@ -195,4 +195,8 @@ def total_price(order_article = self.order_article)
   def result_manually_changed?
     result != result_computed unless result.nil?
   end
+
+  def quantity_with_tolerance
+    tolerance > 0 ? "#{quantity} + #{tolerance}" : "#{quantity}"
+  end
```

Comment:
Are you sure to put this here? In html we may want to include coloring, for example. Maybe keep it in a pdf-related module?